### ISSUE TITLE
repo.iter_frames throws error

### ISSUE THREAD

**Issue by ktugan** [BioroboticsLab/bb_binary#12]
Iterating over frames while using the begin and end parameter may throw an error. The reason for the error could not be found.

I used one hour following data `repo = Repository('data/1h_09-25-15/')`

**code:**
```python
repo = Repository('data/1h_09-25-15/')
frames = repo.iter_frames(begin=ts1, end=ts2)
list(frames)
```

**error**
```
-----------------------------------------------------------------------
AssertionError                        Traceback (most recent call last)
<ipython-input-6-e04f7628f3b5> in <module>()
----> 1 list(frames)

/opt/anaconda2/envs/bee3/lib/python3.5/site-packages/bb_binary/repository.py in iter_frames(self, begin, end, cam)
    212                 FrameContainer (FrameContainer): the corresponding FrameContainer for each frame.
    213         """
--> 214         for f in self.iter_fnames(begin=begin, end=end, cam=cam):
    215             fc = load_frame_container(f)
    216             for frame in fc.frames:

/opt/anaconda2/envs/bee3/lib/python3.5/site-packages/bb_binary/repository.py in iter_fnames(self, begin, end, cam)
    191             else:
    192                 current_path = self._step_to_next_directory(
--> 193                     current_path, direction='forward')
    194                 current_path = self._join_with_repo_dir(current_path)
    195 

/opt/anaconda2/envs/bee3/lib/python3.5/site-packages/bb_binary/repository.py in _step_to_next_directory(self, path, direction)
    306             path_dt = self._get_time_from_path(path)
    307             end_dt = self._get_time_from_path(end)
--> 308             assert path_dt < end_dt
    309 
    310         elif direction == 'backward':

AssertionError: 
```

**Comment by aschle** [comment]
+1

**Comment by aschle** [comment]
Is it possible to fix this? 
It is very annoying and I could not figure out, why it happens.

Maybe some person who knows a bit more, we can sit together and fix this!

**Comment by nebw** [comment]
Fixed in 5379eef
